pkg/slack/filter.go

```diff
@@ -0,0 +1,84 @@
+package slack
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/oursky/github-actions-manager/pkg/github/jobs"
+	"k8s.io/utils/strings/slices"
+)
+
+type messageFilterRule struct {
+	conclusions []string
+	// branches    []string
```

Comment:
Since it indicates a guaranteed future direction for as of then unwritten code I considered it to be at the same level as true code. I will have to change this habit in future. :doglaugh: